instagram/src/components/Login/login.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+
+class Login extends React.Component {
+  constructor() {
+    super();
+    this.state = { username: "", password: "" };
+  }
+
+  changeHandler = e => {
+    e.preventDefault();
+    this.setState({
+      [e.target.name]: e.target.value
+    });
+  };
+
+  login = e => {
+    e.preventDefault();
+    localStorage.setItem("username", this.state.username);
+    window.location.reload();
+  };
+
+  render() {
+    return (
+      <>
+        <input
+          name="username"
+          placeholder="username"
+          value={this.state.username}
+          type="text"
+          onChange={this.changeHandler}
+        />
+        <input
+          name="password"
+          placeholder="password"
+          value={this.state.password}
+          type="text"
+          onChange={this.changeHandler}
```

Comment:
if you want to make sure the fields are filled out before submitting (without having to write your own conditional checking to see if either field are equal to `""` first) you can add the one-word attribute `required` in both of these fields and it does the work for you